src/dashboards/Compact/styles.scss

```diff
@@ -105,9 +105,26 @@
         transition: bottom 0.3s;
     }
 
+    .react-grid-item:matches(#scooter) > .react-resizable-handle::after {
+        content: '';
+        position: absolute;
+        width: 20%;
+        height: 0.25rem;
+        bottom: 0;
+        margin: 0 40%;
+        border-radius: 50vh;
```

Comment:
Det var ekstremt stor border-radius. Kan den vere halvparten av height, kanskje?

Reply:
> Kan den vere halvparten av height, kanskje?
Er dette en generell lifehack for å få perfekt avrunda kanter? :squint: Det funka hvert fall perfekt med halvparten av height 😁